_guides/php/command-line.md

```diff
@@ -133,12 +133,12 @@ The test for class `Greeting` is only verifying the return value of one method.
 
 ### Running the Application
 
-To run the application execute `bin/hello.php "World"` or `php hello.php "World"`.
-You should see the text &ldquo;Hello World&rdquo; being printed.
+To run the application execute `bin/hello.php` or `php bin/hello.php`.
+You should see the text &ldquo;Hello&rdquo; being printed.
```

Comment:
Minor thing I didn't spot before: I think this line can be removed as the last line of the guide says the same.

Reply:
Well spotted!